app/config/config_prod.yml

```diff
@@ -25,3 +25,7 @@ monolog:
             level: debug
         console:
             type:  console
+
+doctrine:
+    orm:
+        metadata_cache_driver: apc
```

Comment:
I've just pushed a change to use apc only if the extension is available.
Wonder if we should link this behavior with the current cache system (_PS_CACHE_ENABLED_ + _PS_CACHING_SYSTEM_), or leave it as is.